app/assets/javascripts/backbone/views/question_group_dialog_view.js.coffee

```diff
@@ -0,0 +1,18 @@
+class ELMO.Views.QuestionGroupDialogView extends Backbone.View
+
+  initialize: ->
+    questionGroup()
+
+  questionGroup = ->
+    $('.q-addGroup').on 'click', (link) =>
+      displayDialog()
+    $('.add-group-cancel').on 'click', (button) =>
+      hideDialog()
+    $('.add-group-save').on 'click', (button) =>
+      hideDialog()
+
+  displayDialog = ->
+    $('#addQuestionGroupModal').css('display', 'block').css('opacity', 1).css('padding-top', '200px')
```

Comment:
This should look something like `$('#mymodal').modal('show')`. Modal styles are already defined and you won't need to do this stuff manually.

See print_form_view.js.coffee line 45, _print_format_tips_modal.html.erb, and the docs on Bootstrap modals for more info.